nemoguardrails/actions/llm/utils.py

```diff
@@ -50,7 +51,17 @@ async def llm_call(llm: BaseLanguageModel, prompt: Union[str, List[dict]]) -> st
             [ChatPromptValue(messages=messages)], callbacks=logging_callbacks
         )
 
-        return result.generations[0][0].text
+        # Return string if there is only one completion,
+        # otherwise, return all completions in a list
+        if len(result.generations[0]) > 1:
+            all_completions = []
+            i = 0
+            while i < len(result.generations[0]):
+                all_completions.append(result.generations[0][i].text)
+                i += 1
+            return all_completions
+        else:
```

Comment:
Here you have an extra tab and the function now returns `None` when the `isinstance(prompt, str)`.